src/features/maintenance/components/kanban-column.tsx

```
@@ -0,0 +1,68 @@
+        onDragOver={e => {
+          e.preventDefault();
+          e.dataTransfer.dropEffect = 'move';
+        }}
+        onDrop={e => {
+          e.preventDefault();
+          const requestId = e.dataTransfer.getData('text/plain');
+          if (onStatusChange && requestId) {
+            onStatusChange(requestId, status);
+          }
+        }}
+      >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)